testing/cli/test_cli.py

```diff
@@ -18,12 +19,13 @@
 
 @pytest.mark.parametrize("script", scripts_to_test)
 def test_calling_scripts_with_no_args_shows_usage(capsys, script):
-    """Test that SCT's scripts all return error code 2 and
-    show usage descriptions when called with no arguments."""
-    mod = importlib.import_module(f"spinalcordtoolbox.scripts.{script}")
-    with pytest.raises(SystemExit) as system_err:
-        mod.main(argv=[])
-    captured = capsys.readouterr()
-
-    assert system_err.value.code == 2
-    assert 'usage' in captured.err.lower()
+    """
+    Test that SCT's scripts all return error code 2 and show usage descriptions when called with no arguments.
+    Also, ensure that calling the help takes under 2.0 seconds per script.
+    """
+    start_time = time.time()
+    completed_process = subprocess.run([script], capture_output=True)
+    duration = time.time() - start_time
+    assert completed_process.returncode == 2
+    assert b'usage' in completed_process.stderr
+    assert duration < 2.0, f"Expected '{script} -h' to execute in under 2.0s, but took {duration}"
```

Comment:
If this test becomes flaky in the future, we can probably just raise the threshold, so I'm not too worried about it.

Reply:
I've added a command here to re-run failed tests, that also [plays nicely with parametrized tests](https://docs.pytest.org/en/latest/how-to/cache.html#rerunning-only-failures-or-failures-first), too. That way, the `2.0s` threshold will only trigger an error if a script takes longer than 2 seconds in back-to-back runs.

I figure that this is a little more useful than increasing the threshold? Since it keeps a more strict goal for our scripts. (I still want the test to trip if a contributor adds a non-LazyLoaded component that slows down a script.)